charts/prometheus-thanos/values.yaml

```diff
@@ -106,7 +106,10 @@ storeGateway:
   # - name: GOOGLE_APPLICATION_CREDENTIALS
   #   value: /etc/gcp/secrets/credentials.json
   logLevel: info
-  indexCacheSize: 500MB
```

Comment:
I'd like to see the config like this:

```
storeGateway:
  indexCache:
    config: 
    type: 
```
Also an example for memcached would be nice.

Reply:
I think a full example would blow up the table a bit, so instead, I linked the Thanos docs in the comment which has a memcached example.